src/platforms/common/configuration/options.mdx

```diff
@@ -89,7 +89,9 @@ By default the SDK will try to read this value from the `SENTRY_ENVIRONMENT` env
 
 <PlatformSection supported={["unity"]}>
 
-Sets the environment. This string is freeform and set by default. A release can be associated with more than one environment to separate them in the UI (think `staging` vs `prod` or similar). Depending on `Application.isEditor` the environment will be set to `editor` or `production` respectively.
+Sets the environment. This string is freeform and set by default. A release can be associated with more than one environment to separate them in the UI (think `staging` vs `prod` or similar).
```

Comment:
I'm not sure if I'm doing this correctly here. Just one big copy-pasta. It's just the difference between "set by default" and "NOT set by default". Is there any better way to handle this?

Reply:
I think we could just add this at the end, and keep the other content available to all platforms (not exclude for Unity).